modules/ossm-config-control-plane-infrastructure-node.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-config-control-plane-infrastructure-node_{context}"]
+== Configuring all {SMProductShortName} Control Plane components to run on infrastructure nodes
```

Comment:
```suggestion
== Configuring all {SMProductShortName} control plane components to run on infrastructure nodes
```

looks like this is lowercase in the below paragraph, so if it's not an official name I'd keep it lowercase for consistency

Reply:
done